frontend/src/components/SideBar/AddGroupTags.module.scss

```diff
@@ -7,8 +7,14 @@
 }
 
 .ClassItem {
+  background: none;
+  border-radius: 0 !important;
   display: flex;
-  padding: 0 0.5em;
+  font-size: 100%;
+  height: 34px !important;
+  margin: 0 !important;
+  padding-left: 10px;
+  width: 100% !important;
```

Comment:
Is there a way we can avoid these `!important`? Maybe we can increase the css specificity for this. Not push blocking at this moment.

Reply:
My rationale is in the `Notes` section, but to be fairly honest it would take me a nontrivial amount of time to try and increase the specificity. At the expense of this slightly suboptimal engineering decision, maybe it's best to move fast since we're sunsetting this sem.